File: C/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    //loop input
    do
    {
        n = get_int("Height: ");
    }
    //Make input from the user in range 1-8
    while (n < 1 || n > 8);

    //this will take care of the amount of rows
    for (int i = 0; i < n; i++)
    {
        //this will make the image backwards
        for (int k = n - 1; k > i; k--)
        {
            printf(" ");
        }
        //this will take care of adding a new "#" every new line
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}

File: sentimental-mario-less/mario.py
# loop user input until number is in range(1,8)
while True:
    try:
        height = int(input("Height: "))
        # break when correct range
        if height >= 1 and height <= 8:
            break
    except:
        pass
# 3 loops, one for making each row, one for making the image backwards, and one for hashes
for i in range(1, height + 1):
    for j in range(height - i):
        print(" ", end="")
    for j in range(i):
        print("#", end="")
    # print here so we get a new line
    print()